lib/grandpa/message_handler.go

```diff
@@ -246,13 +246,35 @@ func (h *MessageHandler) verifyCatchUpResponseCompletability(prevote, precommit
 	return nil
 }
 
+func getEquivocatoryVoters(votes []AuthData) map[ed25519.PublicKeyBytes]struct{} {
```

Comment:
can you make this take `voters []ed25519.PublicKeyBytes` instead? as the `AuthData.Signature` field is unused

Reply:
@EclesioMeloJunior just reminding on this

Reply:
are you going to do this @EclesioMeloJunior ?